### ISSUE TITLE
improper command pipelining after QUIT with postfix 2.11.3-1 (Debian 8.0)

### ISSUE THREAD

**Issue by silviuvulcan** [nagios-plugins/nagios-plugins#145]
Hi,

I recently upgraded to Debian 8 on my mail server and I started getting my nagios server banned by fail2ban. The issue is that the following error appears in the mail.log when check_ssmt runs:

```
$ /usr/local/nagios/libexec/check_ssmtp -H mail.example.com -p 465
```

And this appears in **mail.log**:

```
Mar 28 11:04:40 example postfix/smtpd[9460]: connect from 4-3-2-1.example.com[1.2.3.4]
Mar 28 11:04:40 example postfix/smtpd[9460]: improper command pipelining after QUIT from 4-3-2-1.example.com[1.2.3.4]:
Mar 28 11:04:40 example postfix/smtpd[9460]: disconnect from 4-3-2-1.example.com[1.2.3.4]
```

I updated to nagios plugins 2.1.1 but the only change is that now it only happens intermittently.

**Comment by tmcnag** [comment]
Forum post for reference:

https://support.nagios.com/forum/viewtopic.php?f=35&t=37726#p177449

**Comment by jfrickson** [comment]
I looking into this a couple months ago, and couldn't find a problem or reproduce. I'll be checking it out again soon.

**Comment by jfrickson** [comment]
@silviuvulcan could you run the command line again, and add `-vvv` like so:
```
$ /usr/local/nagios/libexec/check_ssmtp -H mail.example.com -p 465 -vvv
```
then post the output. Maybe that will provide more information.

Thanks

**Comment by silviuvulcan** [comment]
I tried but can't reproduce the error anymore. It seems I upgraded to nagios plugins 2.2.0 some time in the past and maybe postfix was updated on the server (it's a debian 8 kept up-to-date so if any patches were released in the debian repos they were applied). So I can't tell which fixed it. 

I grep-ed the logs but can't find the error associated with a nagios check either so it's been fixed for a while now.
